src/conventions.jl

```diff
@@ -0,0 +1,13 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+Ordering convention used to construct k-faces of a polytope from vertices (0-faces).
+"""
+abstract type Ordering end
+
+"""
+Connectivity function provided by an `Ordering` subtype used to construct k-faces of a polytope from vertices (0-faces).
+"""
+function connectivity end
```

Comment:
Please configure your IDE to not remove the last EOL. VS Code does that by default unfortunately.